src/components/Incorporation/SummaryDefineCompany.vue

```diff
@@ -15,39 +15,41 @@
         </span>
       </div>
 
-      <!-- Name -->
-      <article class="section-container">
-        <v-row no-gutters>
-          <v-col cols="12" sm="3" class="pr-4">
-            <label id="company-label">Name</label>
-          </v-col>
-          <v-col cols="12" sm="9" class="pt-4 pt-sm-0">
-            <div id="company-name">{{ getNameRequestApprovedName || '[Incorporation Number] B.C. Ltd.' }}</div>
-            <div id="company-description">{{ entityDescription }}</div>
-          </v-col>
-        </v-row>
-
-        <!-- Name Translation -->
-        <v-row no-gutters v-if="getNameTranslations && getNameTranslations.length" class="mt-3">
-          <v-col cols="12" sm="3" class="pr-4">
-            <label>Name Translation</label>
-          </v-col>
-          <v-col cols="12" sm="9">
-            <div
-              v-for="(nameTranslation, index) in getNameTranslations"
-              class="text-uppercase"
-              :key="`name_translation_${index}`"
-            >
-              {{ nameTranslation.name }}
-            </div>
-          </v-col>
-        </v-row>
-      </article>
+      <template v-if="showCompanyName">
+        <!-- Name -->
+        <article class="section-container">
+          <v-row no-gutters>
+            <v-col cols="12" sm="3" class="pr-4">
+              <label id="company-label">Name</label>
+            </v-col>
+            <v-col cols="12" sm="9" class="pt-4 pt-sm-0">
+              <div id="company-name">{{ getNameRequestApprovedName || '[Incorporation Number] B.C. LTD.' }}</div>
```

Comment:
To be consistent with everywhere else.

Reply:
^^ this and the related comment below are all very good points.

I have updated this code in a new commit. (Note that the "B.C. LTD." change for ULC and CCC is a separate ticket.)

Reply:
Updated and committed. This code is now on line 142.

Reply:
@jonathan-longe OK to resolve this conversation?